guides/common/modules/con_migrating-from-internal-databases-to-external-databases.adoc

```diff
@@ -20,8 +20,14 @@ endif::[]
 
 To migrate from the default internal databases to external databases, you must complete the following procedures:
 
+ifndef::orcharhino[]
 . xref:preparing-a-host-for-external-databases_{context}[].
```

Comment:
Same here

Reply:
We are trying to display only EL7 in the case of Orcharhino while it redirects to the same process. Also, in the file `proc_preparing-a-host-for-external-databases.adoc` there is only a reference to EL7 for Orcharhino.

This could be the probable reason we used `ifdef` and `ifndef` in the case of Orcharhino. However, I am open to changes :+1:

Reply:
I think what Ewoud correctly suggestions is to "share" the link, but not the other line.

```
line 1 with link
ifdef orcharhino
LINE 2a
endif
ifndef orcharhino
LINE 2b
endif
```

This should avoid duplicating the line with the link.

Reply:
Exactly that.